omise/setting.php

```diff
@@ -0,0 +1,86 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+class Setting extends PaymentModule
+{
+    protected $submit_action = 'omise_save_setting';
```

Comment:
@guzzilar 

> for this case just const make code more understandable that this field will not be changed by any part of code.

Yes, everything has pros and cons. With class constant, it can not be changed and extended.

> can you share the idea behind that you chose protected $submit_action?

The idea is extendability.